src/main/java/tech/fastj/math/Maths.java

```diff
@@ -201,4 +201,15 @@ public static float denormalize(float normalizedNum, float min, float max) {
 
         return (normalizedNum * (max - min)) + min;
     }
+
+    /**
+     * Returns a {@code float} based on the value between a and b.
```

Comment:
This documentation is a bit lacking in description. "The value between a and b" -- for starters, you might want to wrap `a` and `b` with {@code }.
With that in mind, try to write your documentation so that anyone can determine the meaning of each parameter incredibly quickly.